qml/js/youtube/YouTubeCipher.js

```diff
@@ -86,6 +86,6 @@ function decodeCipheredVideo(videoid, mediagrabber, urldecoder)
         }
     }
 
-    req.open("GET", "http://www.youtube.com/watch?v=" + videoid + "&gl=US&persist_gl=1&hl=en&persist_hl=1");
+    req.open("GET", "https://www.youtube.com/watch?v=" + videoid + "&gl=US&persist_gl=1&hl=en&persist_hl=1");
```

Comment:
No problem, I just noticed that this was open.

I am trying to learn coding and I started with Codecademy's JavaScript course and then just went through apps I actively use in case I could do something, but I don't understand QML and nothing in the issue tracker looks like something I could do.